mlflow_elasticsearchstore/elasticsearch_store.py

```diff
@@ -0,0 +1,115 @@
+import uuid
+
+from mlflow.store.tracking.abstract_store import AbstractStore
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE, RESOURCE_ALREADY_EXISTS, \
+    INVALID_STATE, RESOURCE_DOES_NOT_EXIST, INTERNAL_ERROR
+from mlflow.entities import (
+    Experiment, RunTag, Metric, Param, RunData, RunInfo,
+    SourceType, RunStatus, Run, ViewType, ExperimentTag, Columns, LifecycleStage)
+from mlflow.exceptions import MlflowException
+from mlflow.utils.uri import append_to_uri_path
+from elasticsearch_dsl import Index, Search, Q, connections
+
+from models import ElasticExperiment, ElasticRun, ElasticMetric, \
+    ElasticParam, ElasticTag, ElasticLatestMetric
+
+
+class ElasticsearchStore(AbstractStore):
+
+    ARTIFACTS_FOLDER_NAME = "artifacts"
+    DEFAULT_EXPERIMENT_ID = "0"
+
+    def __init__(self, user, password):
+
+        url = user + ":" + password + "@10.236.61.26:80"
+
+        connections.create_connection(hosts=[url])
+
+    def create_experiment(self, name, artifact_location=None):
```

Comment:
Can you add typing for the function parameters and return type ? And to the same for every function

Reply:
You mean adding comment as documentation or is it something else ?

Reply:
No adding typing, you can check the Typing module documentation to know how to do